src/main/scala/io/iohk/ethereum/timing/package.scala

```diff
@@ -0,0 +1,6 @@
+package io.iohk.ethereum
+
+
+package object timing {
+  type SlotNumber = BigInt
```

Comment:
WDYT about for now removing this `type` and just use `BigInt` as with the number in the `BlockHeader`?

Reply:
:+1: